src/scripts/crowdin/import/utils.ts

```diff
@@ -60,6 +58,7 @@ export const scrapeDirectory = (
       // Update .md tracker
       trackers.langs[repoLangCode].mdCopyCount++
     } else {
+      if (!statSync(source).isDirectory()) return
```

Comment:
This `else` block was catching the non-markdown/JSON files (such as `.xlsx`) and attempting to open as a directory; this guard clause will skip non-directories.